dotnet/CoreLib/MemoryStorage/AzureCognitiveSearch/AzureCognitiveSearchMemory.cs

```diff
@@ -109,16 +109,16 @@ public async Task<string> UpsertAsync(string indexName, MemoryRecord record, Can
 
         var client = this.GetSearchClient(indexName);
 
-        SearchQueryVector vectorQuery = new()
+        RawVectorQuery vectorQuery = new()
         {
             KNearestNeighborsCount = limit,
-            Value = embedding.Data.ToArray(),
+            Vector = embedding.Data.ToArray(),
             Fields = { AzureCognitiveSearchMemoryRecord.VectorField }
```

Comment:
could you add `Exhaustive = true`?

we might make it configurable in future, but for now I'd like having the best matches by default